src/site/content/en/progressive-web-apps/offline-fallback-page/index.md

```diff
@@ -0,0 +1,270 @@
+---
+layout: post
+title: Create a simple offline experience
+authors:
+  - thomassteiner
+date: 2020-09-11
+# updated: 2020-09-11
+description: Learn how to add an offline fallback page to your app.
+tags:
+  - progressive-web-apps
+---
+
+What do the Google Assistant app, the Slack app, the Zoom app, the Apple Podcasts app, and almost any other native app on your phone or computer have in common? Right, they never give you nothing. Even when you do not have a network connection, you can still open the Assistant app, or enter Slack, or launch Zoom, or run Podcasts. You might not get anything particularly meaningful or even be unable to achieve what you wanted to achieve, but at least you get *something* and the app is in control.
```

Comment:
>they never give you nothing

Please find a different way to say this. In some dialects of American English this means "they never give you anything." My immediate gut reaction to this sentence is that it was the worst grammar mistake I had ever seen from your writing, even though it's not.

Reply:
Just a quick comment on that one: it’s the negated form of saying “they always (negated never) give you something (nothing)”. 

It’s inspired by [John Grisham](https://www.amazon.com/Firm-John-Grisham/dp/0385416342): “You never get nothing for nothing”.